The PyPI archive doesn't include one file necessary for testing.

This issue was reported upstream:
https://github.com/python/typing_extensions/issues/61

--- /dev/null
+++ typing_extensions-4.3.0/src/_typed_dict_test_helper.py
@@ -0,0 +1,8 @@
+from __future__ import annotations
+
+from typing import Generic, Optional, T
+from typing_extensions import TypedDict
+
+
+class FooGeneric(TypedDict, Generic[T]):
+    a: Optional[T]
